gatsby-config.mjs

```diff
@@ -0,0 +1,83 @@
+import path from "path";
+import { getContentDirectory } from "./getContentDirectory.mjs";
+
+const contentDirectory = getContentDirectory();
+
+export default {
+  siteMetadata: {
+    title: "Recipe Book",
+    siteUrl: "https://recipes.rogermparent.dev",
+  },
+  graphqlTypegen: true,
+  plugins: [
+    "gatsby-plugin-postcss",
+    {
+      resolve: "gatsby-plugin-manifest",
+      options: {
+        name: `Gatsby Recipe Book`,
+        short_name: `Gatsby Recipe Book`,
+        start_url: `/`,
+        background_color: `#f7f0eb`,
+        theme_color: `#a2466c`,
+        display: `standalone`,
+        icon: path.resolve("src", "images", "icon.png"),
+        cache_busting_mode: "none",
+      },
+    },
+    {
+      resolve: "gatsby-plugin-offline",
+      options: {
+        workboxConfig: {
+          runtimeCaching: [
+            {
+              // Use cacheFirst since these don't need to be revalidated (same RegExp
+              // and same reason as above)
+              urlPattern: /(\.js$|\.css$|static\/)/,
+              handler: `CacheFirst`,
+            },
+            {
+              // page-data.json files, static query results and app-data.json
+              // are not content hashed
+              urlPattern: /^https?:.*\/page-data\/.*\.json/,
+              handler: `NetworkFirst`,
+            },
+            {
+              // Add runtime caching of various other page resources
+              urlPattern:
+                /^https?:.*\.(png|jpg|jpeg|webp|svg|gif|tiff|js|woff|woff2|json|css)$/,
+              handler: `NetworkFirst`,
+            },
+            {
+              // Google Fonts CSS (doesn't end in .css so we need to specify it)
+              urlPattern: /^https?:\/\/fonts\.googleapis\.com\/css/,
+              handler: `NetworkFirst`,
+            },
+          ],
+        },
+      },
```

Comment:
Overriding workbox config from `StaleWhileRevalidate` to `NetworkFirst` ensures the editor doesn't hit stale content. There's possibly a better way to do this with manual cache invalidation, but we're more concerned with offline mode than performance and offline mode still works, so this is fine.